client/src/components/App.tsx

```diff
@@ -103,14 +103,16 @@ const App = (): ReactElement => {
         <CssBaseline />
         <Box
           sx={{
+            height: '100vh',
+            scrollMarginBottom: 0,
+            overflowY: 'scroll',
+            overflowX: 'hidden',
```

Comment:
- Does overflowX get rid of any horizontal scroll on every page or is this just background specific?
- Also wondering what sort of effect overflowY has